docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CRD management in ASO
+weight: -3
+---
+
+## How does CRD management work in Azure Service Operator?
+
+ASO performs CRD management differently than many traditional operators. You might notice if you install the
+[Helm chart](../installing-from-helm) or the [YAML](../installing-from-yaml) that there are NO CRDs installed by default.
+
+This is for two reasons:
+
+1. ASO supports so many CRDs that it exceeds the maximum Helm chart size when CRDs are included, so the CRDs cannot be 
+   included in the Helm chart.
+2. ASO supports so many CRDs that it can cause problems if they're all installed onto a Kubernetes cluster with a low
+   memory limit. See [#2920](https://github.com/Azure/azure-service-operator/issues/2920) for details.
+
+For the above reasons, by default each version of the ASO operator pod embeds the CRDs inside the pod and installs them when
+the pod launches. 
+
+**No CRDs are installed by default.** New CRDs will only be installed if they are specified in the 
+`--crd-pattern` cmdline argument to the `manager` container. `--crd-pattern` is a semicolon delimited string of CRD
+patterns to apply. 
+
+**Existing CRDs are always upgraded.** If a CRD already exists in the cluster (from a previous version of ASO or 
+a manual CRD installation), it will always be updated to the latest version of the CRD.
+
+## With Helm
```

Comment:
Is there a better title? I was going to suggest _Helms Based Installations_ but then that's a lot of repetition of _Installation_ in the subheadings.

Reply:
That works. 👍